AntennaTracker/servos.cpp

```diff
@@ -130,12 +130,8 @@ void Tracker::update_pitch_onoff_servo(float pitch)
 */
 void Tracker::update_pitch_cr_servo(float pitch)
 {
-    int32_t pitch_min_cd = g.pitch_min*100;
-    int32_t pitch_max_cd = g.pitch_max*100;
-    if ((pitch>pitch_min_cd) && (pitch<pitch_max_cd)) {
```

Comment:
Shouldn't this be clamping the values rather than doing nothing?

Reply:
i have added the constrain, its abit odd because tracker doesn't use the standard range, https://github.com/ArduPilot/ardupilot/blob/36b2d76cc0655a07da8ae7903c5eb06ffa2c1d8f/AntennaTracker/servos.cpp#L16  I think its ok tho.